main/mc/alpine_syntax.patch

```diff
@@ -1,6 +1,6 @@
---- old/misc/syntax/Syntax.in
-+++ new/misc/syntax/Syntax.in
-@@ -238,6 +238,9 @@
+--- a/misc/syntax/Syntax.in
++++ b/misc/syntax/Syntax.in
+@@ -250,6 +250,9 @@
```

Comment:
this kind of changes does not provide any value and only slow down the review process.

Reply:
@ncopa Noted.